src/ui/drawing/gdi/GDIBitmapSurface.hh

```diff
@@ -89,9 +89,9 @@ public:
     GDIAlphaBitmapSurface& operator=(GDIAlphaBitmapSurface&&) noexcept = delete;
 
     void FillRectangle(int nX, int nY, int nWidth, int nHeight, Color nColor) noexcept override;
-    void WriteText(int nX, int nY, int nFont, Color nColor, const std::wstring& sText) override;
+    void WriteText(int nX, int nY, gsl::index nFont, Color nColor, const std::wstring& sText) override;
 
-    void Blend(HDC hTargetDC, int nX, int nY) const;
+    void Blend(HDC hTargetDC, std::ptrdiff_t nX, std::ptrdiff_t nY) const;
```

Comment:
`nX` and `nY` should have a consistent data type. you've changed this to `ptrdiff_t`, but not `FillRectangle` or `WriteText` (probably others). 

is there a better type for coordinates? there's no pointers involved.

Reply:
There's `intptr_t`, or should we just make a `Point` structure? Or maybe alias Point from Position in types.hh. It does make sense to use to use int, I'll see if I can do that without narrowing, I do think pairs can abstracted.

`using Point = Position;`